lib/config/config.go

```diff
@@ -203,6 +215,14 @@ func readFileToConfig(pathToConfig string) (*Config, error) {
 		config.FlushSizeKb = defaultFlushSizeKb
 	}
 
+	if config.Mode == "" {
```

Comment:
This statement is duplicated.

Reply:
Thx

Reply:
Could also throw an error if the mode is invalid (i.e. not "history" or "replication").

Reply:
I think doing that for a later version would make more sense